src/CommandObserver.ts

```diff
@@ -15,6 +15,16 @@ export class CommandObserver {
     private _diagnosticCollection: vscode.DiagnosticCollection = null;
     private _diagnosticMap = new Map<string, vscode.Diagnostic[]>();
 
+	private _buildInProgress: boolean = false;
```

Comment:
Tabs vs spaces seems to be a common problem...

Reply:
I checked across azure data studio and other extensions and tabs is used. So keeping it tabs to be consistent.

Reply:
This project was using spaces. If you contribute to a project you should follow their guidelines. :\